### ISSUE TITLE
x64 XAML builds fail in Visual Studio 2017

### ISSUE THREAD

**Issue by AndyGerlicher** [Microsoft/msbuild#1943]
1. Create a new Workflow project (e.g. Activity Library).
2. Change `Platform` to x64 or build with x64 version of MSBuild.
3. Note build failure:
```
XC1020: Build error occurred in the XAML MSBuild task: 'Could not load file or assembly 'XamlBuildTask, Version=15.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35' or one of its dependencies. The system cannot find the file specified.'	ActivityLibrary1
```
Or from command-line:
```
MSB4062: The "ReportDeferredValidationErrorsTask" task could not be loaded from the assembly Microsoft.Activities.Build, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35. Could not load file or assembly 'Microsoft.Activities.Build, Verrion=15.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35' or one of its dependencies. The system cannot find thefile specified. Confirm that the <UsingTask> declaration is correct, that the assembly and all its dependencies are available, and that the task contains a public class that implements Microsoft.Build.Framework.ITask.
```

**Comment by smad2005** [comment]
How i understand fix will come in 15.3?

**Comment by CaryFan** [comment]
This problem was not fixed at least in the MSBuild 15.0 which were downloaded by me at the end of 07.2017. This is very irritating. I ran across this problem in command prompt when directly using MSBuild to build my code base. Instead, if I build the code in VS2017, it works quite well.

**Comment by phillip-haydon** [comment]
Same issue here... Can't run the unit tests in ncrunch or resharper cos of this issue.
